x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/assembler/ImplicitTiebreakerTests.java

```diff
@@ -65,7 +65,7 @@ public void query(QueryRequest r, ActionListener<SearchResponse> l) {
                 int previous = ordinal - 1;
                 // except the first request, the rest should have the previous response's search_after _shard_doc value
                 assertArrayEquals("Elements at stage " + ordinal + " do not match",
-                    r.searchSource().searchAfter(), new Object[] { (long) previous, implicitTiebreakerValues.get(previous) });
+                    r.searchSource().searchAfter(), new Object[] { String.valueOf(previous), implicitTiebreakerValues.get(previous) });
```

Comment:
Leftover from undo-ing the implicit tiebreaker changes?

Reply:
This has be a string, since otherwise the value - before a Long/Instant, now a Timestamp - can't be used when processing downstream the search object.